### PR TITLE
Updates to some rails conventions

### PR DESCRIPTION
**Pull request by yodarjun**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,7 +9,6 @@ It was inspired by Airbnb's styleguide.
     1. [Indentation](#indentation)
     1. [Inline](#inline)
     1. [Newlines](#newlines)
-    1. [Blocks](#blocks)
```
Comment (yodarjun) [inline README.md:12]: this was redundant. it was covered elsewhere

File: README.md [hunk 2]
```diff
@@ -427,7 +388,7 @@ In either case:
       add_trebuchet_experiments_on_page(request_opts[:trebuchet_experiments_on_page])
     end
 
-    # bad - this is complex and deserves multiple lines and a comment
+    # bad - this is complex and deserves multiple lines
```
Comment (benseligman) [inline README.md:391]: :+1:

File: README.md [hunk 3]
```diff
@@ -539,7 +500,7 @@ In either case:
 
 ## Syntax
 
-* <a name="no-for"></a>Never use `for`, unless you know exactly why. Most of the 
+* <a name="no-for"></a>Never use `for`, unless you know exactly why. Most of the
```
Comment (benseligman) [inline README.md:503]: we could probably just do a blanket never use for. i've never once seen it in real ruby code

**Comment by michaeldauria** [comment]
:+1:

File: README.md [hunk 4]
```diff
@@ -575,6 +536,12 @@ In either case:
     # good
     names.select { |name| name.start_with?("S") }.map { |name| name.upcase }
 
+    # good
+    names.each do |name|
+      puts name
+      do_something_with name
```
Comment (benseligman) [inline README.md]: `do_something_with(name)`

File: README.md [hunk 5]
```diff
@@ -1109,6 +1076,24 @@ In either case:
     scope :foo, -> { where(:bar => 1) }
     ```
 
+### Dates
```
Comment (benseligman) [inline README.md:1102]: is this a ruby thing or a rails thing?

**Comment by benseligman** [comment]
:+1: otherwise

File: README.md (hunk 5 above)
Comment (yodarjun) [inline README.md:1102]: This is a rails thing

File: README.md (hunk 3 above)
Comment (yodarjun) [inline README.md:503]: i think its good to have in the doc at least to know the difference between for and each

File: README.md (hunk 5 above)
Comment (benseligman) [inline README.md:1102]: so should that be in this pull then?

File: README.md (hunk 3 above)
Comment (benseligman) [inline README.md:503]: sg

**Pull request closed without merging**